src/components/Button/Button.tsx

```diff
@@ -124,6 +126,7 @@ export const Button: FC<ButtonProps> = ({
                           ]
                         : [isFocusVisible && FOCUS_STYLE, styles[getButtonTheme()][style]],
                 ),
+                hugWidth && "tw-w-full",
```

Comment:
```suggestion
                !hugWidth && "tw-w-full",
```
"Hug" is the current behavior (inline & fit to content)

Reply:
Ah sorry, then I misunderstood. Thank you for clarifying.